File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/DreamwroughtHide.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item;

/* JADX INFO: loaded from: classes3.dex */
public class DreamwroughtHide extends Item {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.item_dreamwrought_hide_name;
        this.idDescription = R.string.item_dreamwrought_hide_description;
        this.idImage = R.drawable.dreamwrought_hide;
        this.source.add(Integer.valueOf(R.string.raid_name_sleeping_planet));
        this.price = 25L;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/DreamwroughtHide.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item

class DreamwroughtHide : Item() {
    override fun configureProperties() {
        idName = R.string.item_dreamwrought_hide_name
        idDescription = R.string.item_dreamwrought_hide_description
        idImage = R.drawable.dreamwrought_hide
        source.add(R.string.raid_name_sleeping_planet)
        price = 25L
    }
}
